syntax = "proto3";

service Firefly {
  rpc GetPhase (PhaseRequest) returns (PhaseResponse);
}

message PhaseRequest {
  int32 id = 1;  // ID des anfragenden Gl端hw端rmchens
}

message PhaseResponse {
  float phase = 1;  // Aktuelle Phase des Gl端hw端rmchens
}
